gcp-py-gke/__main__.py

```diff
@@ -19,11 +19,13 @@
 # password is the password for the admin user in the cluster.
 PASSWORD = config.get_secret('password') or RandomString("password", length=20, special=True).result
 
+engine_version = Output.from_input(get_engine_versions()).latest_master_version
```

Comment:
This is pretty "strange" looking code to have to add to the top of many of our intro-to-pulumi examples.  I'd actually be inclined to just hard code a version and have to fix up our examples occasionally as these get updated - or pick this up from config and hard code the values into the test config?

I don't love hurting readability/usability of examples just to make testing easier.

Reply:
I'd love to make this easier to read, but I do kind of prefer this approach to the hard-coded version approach. If I knew Python better, I would have avoided the `Output.from_input` bit.

That said, this is important content, so I'll defer to you if you feel strongly here.

Reply:
Hard-coding into the config seems reasonable-ish, as we could also pull the latest version inside the test framework if we cared to.